frontend/admin/src/js/components/user/ViewUser.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import {
+  HomeIcon,
+  DownloadIcon,
+  UserCircleIcon,
+} from "@heroicons/react/outline";
+import Breadcrumbs from "@common/components/Breadcrumbs";
+import type { BreadcrumbsProps } from "@common/components/Breadcrumbs";
+import PageHeader from "@common/components/PageHeader";
+import { Button } from "@common/components";
+import { Tab, TabSet } from "@common/components/tabs";
+import { commonMessages } from "@common/messages";
+import { useAdminRoutes } from "../../adminRoutes";
+import { User, useUserQuery } from "../../api/generated";
+import DashboardContentContainer from "../DashboardContentContainer";
+
+interface ViewUserPageProps {
+  user: User;
+}
+
+export const ViewUserPage: React.FC<ViewUserPageProps> = ({ user }) => {
+  const intl = useIntl();
+  const adminPaths = useAdminRoutes();
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Candidate Details",
+    description: "Title for the page when viewing an individual user.",
+  });
+
+  const userName = `${user?.firstName} ${user?.lastName}`;
+
+  const links = [
+    {
+      title: intl.formatMessage({
+        defaultMessage: "Home",
+        description: "Breadcrumb title for the home link.",
+      }),
+      href: adminPaths.home(),
+      icon: <HomeIcon style={{ width: "1rem", marginRight: "5px" }} />,
+    },
+    {
+      title: intl.formatMessage({
+        defaultMessage: "Users",
+        description: "Breadcrumb title for the all users table link.",
+      }),
+      href: adminPaths.userTable(),
+      icon: <UserCircleIcon style={{ width: "1rem", marginRight: "5px" }} />,
+    },
+    {
+      title: userName,
+    },
+  ] as BreadcrumbsProps["links"];
+
+  return (
+    <>
+      <PageHeader icon={UserCircleIcon}>{pageTitle}</PageHeader>
+      <Breadcrumbs links={links} />
+      <div
+        data-h2-align-items="b(center)"
+        data-h2-display="b(flex)"
+        data-h2-flex-direction="b(column) m(row)"
+        data-h2-margin="b(top-bottom, l)"
+      >
+        {userName !== " " && (
+          <h2
+            data-h2-margin="b(top-bottom, s) m(top-bottom, none)"
+            data-h2-font-weight="b(800)"
+          >
+            {userName}
+          </h2>
+        )}
+        <div data-h2-margin="m(left, auto)">
+          <Button
+            mode="outline"
+            color="primary"
+            type="button"
+            onClick={() => {
+              return null;
+            }}
+          >
+            <span>
+              <DownloadIcon style={{ width: "1rem" }} />{" "}
+              {intl.formatMessage({
+                defaultMessage: "Download Profile",
+                description: "Text for button to download a user",
+              })}
+            </span>
+          </Button>
```

Comment:
From a accessibility standpoint I'm pretty sure it's better to use a <Link /> in this situation. It should also include the [download attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download), which can be set to something like ` download="<username>-profile.<filetype>"` (but for now leaving it blank is fine since this is just the placeholder issue). The href can also be set to `href="#"` for now.

Reply:
check